test/kuttl/tests/change_neutron_config/03-assert.yaml

```diff
@@ -0,0 +1,5 @@
+# check that now, debug is not set in neutron config
+apiVersion: kuttl.dev/v1beta1
+kind: TestAssert
+commands:
+  - script: $NEUTRON_KUTTL_DIR/../common/scripts/check_debug_in_neutron_pod_logs.sh --reverse
```

Comment:
I'd suggest to not make determination of what to check here; instead, leave the decision to the caller. you should be able to put `script` command as follows:

for test with debug=false:

`test $(check_debug_in_neutron_pod_logs.sh) = 0`

for test with debug=true:

`test $(check_debug_in_neutron_pod_logs.sh) != 0`

Then you can remove --reverse flag; and you can just return the result of `grep` from your script.

Reply:
To reflect discussion elsewhere, you should use `-ne` and `-eq` since these are integers.